.banzaicloud/icon.svg

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
could you run this through [svgo](https://jakearchibald.github.io/svgomg/)?

Reply:
Sure